src/screens/NationDetailsContainer/index.js

```diff
@@ -124,17 +184,23 @@ class NationDetailsContainer extends NavigatorComponent {
         {...this.props}
         joinNation={this.onJoinNation}
         leaveNation={this.onLeaveNation}
-        deleteDraft={this._onDeleteDraft}
-        submitDraft={this._onSubmitDraft}
+        deleteDraft={this.onDeleteDraft}
+        submitDraft={this.onSubmitDraft}
         openNationChat={this.openNationChat}
       />
     );
   }
 }
 
-NationDetailsContainer.PropTypes = {
-  navigator: PropTypes.object,
-  isDraft: PropTypes.bool,
+NationDetailsContainer.defaultProps = {
+  isDraft: null,
```

Comment:
Please don't pass null to props that has required type (boolean in that case). If there is no correct default value, you can just remove it from there.

Reply:
Done